src/components/ListPaginationCounter.tsx

```diff
@@ -0,0 +1,22 @@
+import { Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+interface ListPaginationCounterProps {
+  start: number;
+  end: number;
+  total: number;
+}
+
+export const ListPaginationCounter = ({ start, end, total }: ListPaginationCounterProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Typography aria-live="polite">
```

Comment:
er ikke aria-live="polite" noe man kun skal trenge å bruke hvis noe tar lang tid å laste inn? Spør altså, tror jeg har misforstått noe.

Reply:
Må inrømme at jeg har mindlessly copy pastet denne. Men når jeg ser på mdn docs så er det kanskje riktig å ha den? "Viser x-y av z" er jo noe som oppdaterer seg når leser paginerer, så det at det er en polite aria-live er kanskje riktig? Litt usikker, mulig @elkebab må svare.

Use polite if:
> A status bar or area of the screen that provides an updated status of some kind. Screen reader users have a special command to read the current status.

Går sidetreff inn under denne?

Reply:
Merger for å slippe hassle med videre jobbing på komponenten